integration-tests/tests/e2e.rs

```diff
@@ -114,6 +124,8 @@ where
     F: Fn(&TestContext) -> Result<(&'static str, String, &'static str)>,
 {
     let ci = env::var("FDO_PRIVILEGED").is_ok();
+    let per_device = env::var("PER_DEVICE_SERVICEINFO")?;
+    L.l(format!("per_device env variable:\n {}", per_device));
```

Comment:
To make things clearer, we should fail the test here if `PER_DEVICE_SERVICEINFO` is set to true, since this test has become the non-per-device service info test.
Please rename this function to `test_e2e_impl_standard_serviceinfo` or something like that to better differentiate the functions

also nit: ` L.l(format!("per_device env variable:\n {per_device}"));`

Reply:
actually I printed this statement during test to check is default env is set to false or not . But yeah the way we set env::set_var("PER_DEVICE_SERVICEINFO", "true"); at the start of test_per_device_serviceinfo() , it will make sense to set it to false at the start of test_e2e_impl_standard_serviceinfo() test. 
And will rename the test too.